.eslintignore

```diff
@@ -7,6 +7,7 @@ node_modules/
 dist
 input
 assets
+coverage
```

Comment:
Added `coverage` folder to ignore as I was using the coverage report to verify the PR's claims while also checking for any loose ends.